NDB.Covid19/NDB.Covid19.iOS/Views/ErrorStatus/ErrorPageViewController.cs

```diff
@@ -107,8 +107,17 @@ private void SetupStyling()
             {
                 ErrorMessageLabel.AccessibilityLabel = ErrorViewModel.REGISTER_ERROR_ACCESSIBILITY_TOOMANYTRIES_DESCRIPTION;
             }
-        }
 
+
+			StyleUtil.InitButtonStyling(ContinueWithSelftestButton, ErrorViewModel.REGISTER_SELFTEST_BUTTON);
+
+			if (ErrorTitle != ErrorViewModel.REGISTER_ERROR_NOMATCH_HEADER)
+            {
```

Comment:
Fix indent

Reply:
what do you mean?

Reply:
Allign `{ `so it is placed right under the `if`, same indentation as on the line above

Reply:
yes i did that now.

Reply:
looks like indentation is broken in the whole file, if you have time you can fix, if not, just merge